clients/admin-ui/src/features/common/custom-fields/CreateCustomLists.tsx

```diff
@@ -8,9 +8,9 @@ import { useAlert } from "~/features/common/hooks";
 import { useUpsertAllowListMutation } from "~/features/plus/plus.slice";
 import { AllowListUpdate } from "~/types/api";
 
+import { TrashCanSolidIcon } from "../Icon/TrashCanSolidIcon";
```

Comment:
nit: in general, we prefer not to do much relative imports. I'd say in the same folder is okay (i.e. `./component`) but once we're going to different folders like `../`, it starts to get confusing when you have to move components around. this could be an absolute import instead, i.e. `~/features/common/Icon/...` which means when you move the file the imports don't have to change.

but sometimes your IDE will import things for you relatively instead of absolutely and it'll sneak up on you 😅

Reply:
ok I'll update this